etcd3/locks.py

```diff
@@ -55,6 +61,9 @@ def acquire(self, timeout=10):
         :returns: True if the lock has been acquired, False otherwise.
```

Comment:
Suggest to add optional param `raise_on_timeout` and update thre "returns" description.